src/compiler/binder.ts

```diff
@@ -253,7 +251,9 @@ namespace ts {
             Debug.attachFlowNodeDebugInfo(reportedUnreachableFlow);
 
             if (!file.locals) {
+                tracing?.push(tracing.Phase.Bind, "bindSourceFile", { path: file.path }, /*separateBeginAndEnd*/ true);

```

Comment:
I considered renaming this but I'm pretty sure the old tracepoint had no consumers and it's a good  name.